test/spartan_port_parse_test.erl

```diff
@@ -0,0 +1,20 @@
+%%%-------------------------------------------------------------------
+%%% @author grubio
+%%% @copyright (C) 2016, <STRATIO>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 03. ago 2016 14:12
+%%%-------------------------------------------------------------------
+-module(spartan_port_parse_test).
+-author("grubio").
+
+-include_lib("eunit/include/eunit.hrl").
+
+port_include_test() ->
```

Comment:
These are eunit tests. It probably makes sense to keep these in the same file (spartan_app.erl)